Classes/FlowBootstrapTrait.php

```diff
@@ -50,13 +53,23 @@ final protected static function bootstrapFlow(): Bootstrap
         if (!defined('BEHAT_ERROR_REPORTING')) {
             define('BEHAT_ERROR_REPORTING', E_ALL);
         }
```

Comment:
but we do `require_once dirname(__DIR__, 4) . '/Packages/Framework/Neos.Flow/Classes/Core/Bootstrap.php';` above ... this is wrong no?

https://github.com/neos/BuildEssentials/pull/71#discussion_r1646163303

Reply:
Note I think it would be more sensible to require the composer autoloader and give the instance to the flow bootstrap as second parameter, that is already possible and IMHO the cleanest way.